standard/docs/en/implementation/amendments.md

```diff
@@ -16,21 +16,43 @@ The nature of a change can be made explicit using:
 
 * **The amendments** building block. This can contain an array of amendment explanations, and clearly identify the releases that contain before and after values. 
 
-## Worked example
+## Worked examples
 
-In the example below:
+Each example includes sample JSON files located at the bottom of the page. Use the combobox at the end of the page to display the files for each example and step.
 
-* A tender is issued for a "Data merge tool" in a release with the `tag` 'tender'. 
-* This is updated in a release with the `tag` 'tenderUpdate' to provide an expanded description of the item. The publisher did not consider this to be a formal 'amendment' to the tender, so did not publish any amendment information.
-* The tender is then formally amended, with an increased value. This is published in a release with the `tag` 'tenderAmendment', and an `amendments` block under `tender`.
+### Example 1: Tender
 
+This example goes through updates during the **tender** stage in a contracting process.
+
+1. A tender is issued for a "Data merge tool" in a release with the `tag` 'tender'. 
+2. Information is updated in a new release with the `tag` 'tenderUpdate' to provide an expanded description of the item. The publisher did not consider this to be a formal 'amendment' to the tender, so did not publish any amendment information.
+3. Then there is a new update, where the value of the tender increases and the deadline for bid submissions is extended. This is considered as an 'amendment' by the publisher (depending of jurisdiction, both changes require an official approval), and so the new release has the `tag` 'tenderAmendment' and an `amendments` block under `tender`. The release reflects the updated value (USD 2000 instead of USD 1000) and the updated closing date for bid submissions (`2012-02-20` instead of `2012-02-15`).
+
+Note in this example that:
+
+* **The amendments block does not contain amended values (as in previous versions of OCDS)**. Values are updated in the same fields and using a new release. 
+
+* **The publisher chooses in the tenderAmendment release to repeat a fragment of the original 'tender' release**. This is not necessary when a full archive of releases is made accessible, but can be a convenient way of publishers making structured information on the before and after values of a field more accessible to users. 
+
+* **In the merged record**, the information in the `amendments` array can be used to lookup information in the `versionedRelease` section, to see where changes are explained by an amendment `rationale`. 
```

Comment:
Please update `versioned-tender.json` to reflect the changes in `merge-tender-3.json`

> * **In the merged record**, the information in the `amendments` array can be used to lookup information in the `versionedRelease` section, to see where changes are explained by an amendment `rationale`. 

What information from the amendments array would be looked up in the versionedRelease?

Reply:
This was in the original example as well. `releaseID` and `amendsReleaseID` can be used to find changes in the `versionedRelease`.

Reply:
Do you think that it would make more sense to say that `releaseID` and `amendsReleaseID` can be used to find the releases with the before and after values in the `releases` array?

I am wondering which is easier from a user's perspective: checking through every field in the versioned release to find where both `releaseID` and `amendsReleaseID` appear, or comparing the two releases in the releases array.

Reply:
Did you have a view on this?

If we don't know which is easier, we can just update the sentence to mention both, e.g.

> * **In the merged record**, the information in the `amendments` array can be used to lookup information in the `versionedRelease` section or in other releases, to see where changes are explained by an amendment `rationale`.